CONTRIBUTING.md

```diff
@@ -1,7 +1,68 @@
 # Contributing
 
-Our goal is to provide high-quality names that make intuitive sense. In our experience, this goal is best achieved through contribution of original names, rather than copying names from other mappings projects. While we believe that discussions relating to the names used by other mappings projects can be useful, those names must stand up to scrutiny on their own - we won't accept names on the grounds that they're present in other mappings projects. 
+Our goal is to provide high-quality names that make intuitive sense. In our experience, this goal is best achieved
+through contribution of original names, rather than copying names from other mappings projects.
+While we believe that discussions relating to the names used by other mappings projects can be useful, those names
+must stand up to scrutiny on their own - we won't accept names on the grounds that they're present in other mappings
+projects. 
 
-We recommend discussing your contribution with other members of the community - either directly in your pull request, or in our other community spaces. We're always happy to help if you need us!
+We recommend discussing your contribution with other members of the community - either directly in your pull request,
+or in our other community spaces. We're always happy to help if you need us!
 
 Please have a look at the [naming conventions](/CONVENTIONS.md) before submitting mappings.
+
+## Pull Request Process
+To get a pull request merged into Quilt Mappings, it must get a certain number of approvals from the maintainers,
+and then it will enter a Final Comment Period. If the pull request passes the final comment period without opposition,
+the PR will be merged. Otherwise, the PR will return to being in review.
+
+The exact number of reviews needed, and the length of the Final Comment Period, varies depending on the scope and
+complexity of the pull request. The numbers for each category are listed below.
+
+> **Minor changes such as typo fixes may bypass the process.**
+
+GitHub PR labels should be used to indicate which category a PR belongs to.
+
+### New Mappings
+**Required Approvals**: 3
+- At least 2 from Mappings Triage and 1 from the Mappings Team.
+- Approvals from the Mappings Team are counted as 2 from Mappings Triage, so 2 approvals from the Mappings Team is
+  enough.
+
+**Final Comment Period**: May vary depending on the complexity of the PR and the type of version it targets.
+- Releases: 2 to 5 days
+- Snapshots: 12 hours to 3 days, may be longer if the PR is large.
```

Comment:
I don't like not being able to know how long the FCP for a PR will be until a Mappings Team member places it in the Final Comment Period. I elaborated on my recommended change for this in my main review comment.